src/App.js

```diff
@@ -1,23 +1,16 @@
-import logo from './logo.svg';
+import React from 'react';
+
 import './App.css';
+import Footer from './components/Footer/Footer.js';
+import Main from './components/Main/Main.js';
+import Header from './components/Header/Header.js';
 
 function App() {
   return (
     <div className="App">
```

Comment:
You can add the background image here, within the App div, to span across the entire webpage